code/style.css

```diff
@@ -1,3 +1,182 @@
-body {
-  background: #FFECE9;
-}
\ No newline at end of file
+.body {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	font-family: 'Montserrat', sans-serif;
+	background-color: #0d1017;
+	color: white;
+	max-width: 400px;
+	margin: auto;
+}
+
+.header {
+	text-align: center;
+}
+
+.user-data {
+	display: flex;
+	flex-direction: column;
+	gap: 4px;
+	align-items: center;
+}
+.user-name {
+	font-size: 30px;
+	margin: 0;
+}
+.avatar-user {
+	width: 100%;
+	border-radius: 50%;
+}
+.avatar-user:hover {
+	border: 3px solid #21262c;
+}
+
+.project-header {
+	text-align: center;
+}
+
+.avatar-collaborator {
+	box-sizing: border-box;
+	height: 30px;
+	border-radius: 50%;
+	padding: 0 2px;
+}
+
+.avatar-collaborator:hover {
+	border: 1px solid #21262c;
+}
+
+.filters-sorting-heading {
+	margin: 0;
+}
+
+.filters-sorting {
+	display: flex;
+	gap: 5px;
+	align-items: center;
+	margin: 5px 0;
+}
+
+.filters-sorting button {
+	height: 30px;
+	border: 1px solid #21262c;
+	border-radius: 10px;
+	margin: 0;
+	background-color: #21262c;
+	color: white;
+}
+
+.projects {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	gap: 10px;
+}
+
+.chart {
+	margin: 40px;
+}
+
+.repo-info {
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+	border: 2px solid #30353b;
+	border-radius: 5px;
+	padding: 10px;
+	margin: 0;
+}
+.repo-heading {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+.numbers {
+	display: flex;
+	gap: 4px;
+}
+
+.repo {
+	font-size: 20px;
+	margin: 5px 0;
+}
+
+.lang {
+	border-radius: 10px;
+	color: black;
+	padding: 0 4px;
+}
+.JavaScript {
+	background-color: #f1e15a;
+	border: 1px solid #f1e15a;
+}
+.HTML,
+.CSS {
+	background-color: #e44c27;
+	border: 1px solid #e44c27;
+}
+
+.info,
+.update,
+.commit,
+.size,
+.pull,
+.collaboration {
+	margin: 2px 0;
+	display: flex;
+	align-items: center;
+}
+
+/* unvisited link */
```

Comment:
thank you for these comments. your code is very complex for me so the comments help me understand it better!